README.md

```diff
@@ -160,8 +173,34 @@ Console.WriteLine(example.ToXML());
 ```
 
 ### VRDR.Messaging
+
 This directory contains classes to create and parse FHIR messages used for Vital Records Death Reporting.
 
+#### Usage
+
+This package is published on NuGet, so including it is as easy as:
+```xml
+<ItemGroup>
+  ...
+  <PackageReference Include="VRDR.Messaging" Version="3.1.0-preview3" />
+  ...
+</ItemGroup>
+```
+
+Note that the VRDR.Messaging package automatically includes the VRDR package, a project file should not reference both.
+
+You can also include the library by referencing `VRDRMessaging.csproj` in your project configuration, for example (taken from VRDR.CLI):
```

Comment:
Suggest adding a bit to make the context clear, bolded text shows my suggested addition:

You can also include **a locally downloaded copy of** the library by referencing `VRDRMessaging.csproj` in your project configuration, for example (taken from VRDR.CLI):